### PR TITLE
[lint] Add lint checking to FPV tb

### PR DESCRIPTION
**Pull request by cindychip**
This PR adds lint checking for FPV testbench.

Signed-off-by: Cindy Chen <chencindy@google.com>

### CODE REVIEW

**Review by weicaiyang — APPROVED** [review]
LGTM

File: hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+  // This is the primary cfg hjson for FPV linting. It imports ALL individual lint
+  // cfgs of the IPs FPV environments.
+  // This enables to run them all as a regression in one shot.
+  name: top_earlgrey_fpv_batch
+
+  import_cfgs:      [// common server configuration for results upload
+                     "{proj_root}/hw/data/common_project_cfg.hjson"
+                     // tool-specific configuration
+                     "{proj_root}/hw/lint/tools/dvsim/{tool}.hjson"]
+
```
Comment (msfschaffner) [inline hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson:15]: Can you also add this rel path here and add a link to the [HW dashboard](hw/_index.md)?
```
  // Different dashboard output path for each tool
  rel_path: "hw/top_earlgrey/fpv/lint/{tool}"
```

**Review by msfschaffner — APPROVED** [review]
Thanks for doing this. One comment, otherwise LGTM

File: hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson (hunk 1 above)
Comment (cindychip) [reply hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson:15]: Oh Thanks Michael, that is what this path is for.. Thanks and will do now!

Comment (cindychip) [reply hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson:15]: Do I need to add to some nightly lint regression script?

**Comment by cindychip** [comment]
This PR solves issue : https://github.com/lowRISC/opentitan/issues/3389

File: hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson (hunk 1 above)
Comment (msfschaffner) [reply hw/top_earlgrey/lint/top_earlgrey_fpv_lint_cfgs.hjson:15]: Hey Cindy, yes I'll send you the path to the file directly. It is not in this repository, though.

**Pull request merged**